.travis.yml

```diff
@@ -6,20 +9,26 @@ cache:
 matrix:
   include:
     - node_js: '7'
-      script: npm run pretest
-      env: CI=pretest
+      script: npm run preci
+      env: CI=preci
     - node_js: '7'
-      script: npm run jest -- --runInBand
-      env: CI=tests 7
+      script: npm run jest -- --coverage --runInBand
```

Comment:
Even though this PR is currently passing the Travis CI tests I think this is going to be an issue per #2174 

_FYI: Once I've finished reviewing this PR I'll take another look at #2174 and add another note here_